_docs/kustomize/networking/network-policy-akash-services.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: akash-services-default-deny-ingress
+  namespace: akash-services
```

Comment:
can these be resources of of a top-level `kustomization.yaml`?  It'd be nice to only have the namespace names declared in one spot.

Reply:
Done. Separated the purely declared yamls from the kustomize'd base files with directories.